internal/ld/ld.go

```diff
@@ -389,11 +392,59 @@ func (b BranchRep) TotalHunkCount() int {
 	return count
 }
 
+func (b BranchRep) WriteToCSV(outDir, projKey, repo, sha string) (path string, err error) {
+	tag := b.Name
+	if len(sha) >= 7 {
+		tag = sha[:7]
+	}
```

Comment:
Try to create a filename with a shortened sha, but if the sha is too short for some unexpected reason, use the branch name instead

Reply:
very nit: but I generally tend to prefer putting the unhappy path cases in the if statement and let the happy path cases be the default.